garak/generators/nvcf.py

```diff
@@ -33,7 +33,7 @@ class NvcfGenerator(Generator):
 
     def __init__(self, name=None, generations=10):
         self.name = name
-        self.fullname = f"NVCF {self.name}"
+        self.fullname = f"{self.generator_family_name} {self.__class__.__name__}"
```

Comment:
This should likely still include the passed in `name` of the model:
```suggestion
        self.fullname = f"{self.generator_family_name} {self.__class__.__name__} {self.name}"
```